cave/utils/convert_for_epm.py

```diff
@@ -78,12 +81,24 @@ def convert_data_for_epm(scenario:Scenario, runhistory:RunHistory):
                                         impute_state=[
                                             StatusType.TIMEOUT, ],
                                         imputor=imputor)
-    else:
+        try:
+            X, Y = rh2EPM.transform(runhistory)
+        except RuntimeError as err:
+            if logger:
+                logger.warning("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+                               "There was some error with the runhistory-"
+                               "transformation in the random-forest (run_obj "
+                               "is %s, if runtime then logged data is expected) "
+                               "with error message: %s", run_obj, err)
+                logger.debug(traceback.format_exc())
+            run_obj = 'runtime_'
```

Comment:
Why can this fail? And why change the run_obj to 'runtime_'?
Is this an issue arising from SMAC2 input?

Reply:
see slack. It's something I don't really understand.